index.html

```diff
@@ -1,5 +1,6 @@
 <html>
   <body>
     1. Komenda "git add .": dodaje zmiany z bieżącego katalogu roboczego do stage'a
+       Komenda "git branch "name"": tworzy nowa galaz o podanej nazwie
```

Comment:
Mogłabyś dodać polskie znaki (ą, ł, ź)?

Reply:
Czy muszę dodać numerację?

Reply:
Skoro jest już zmergowane z numeracją to raczej tak. Możesz też w sumie dać tag <br/>, bo w takiej formie będzie to mało czytelne w przeglądarce